src/main/java/by/custom/utilcalculator/service/BundleResourcesServant.java

```diff
@@ -26,7 +19,12 @@ private static class BundleResourcesCreatorHolder {
     }
 
     public String getString(final String stringToGetFromBundle) {
```

Comment:
I think this name is more attractive and informative enough.
```suggestion
    public String getString(final String localizationKey) {
```

Reply:
Renamed.